tools/c7n_azure/c7n_azure/resources/cost_management_export.py

```diff
@@ -0,0 +1,163 @@
+
+    :example:
+
+    Returns all cost exports for current subscription scope
+
+    .. code-block:: yaml
+
+        policies:
+          - name: get-cost-management-exports
+            resource: azure.cost-management-export
+
+    """
+
+    class resource_type(QueryResourceManager.resource_type):
+        doc_groups = ['Cost']
+
+        service = 'azure.mgmt.costmanagement'
+        client = 'CostManagementClient'
+        enum_spec = ('exports', 'list', None)
+        default_report_fields = (
+            'name',
+            'location',
+            'resourceGroup',
+        )
+        resource_type = 'Microsoft.Compute/images'
+
+        @classmethod
+        def extra_args(cls, resource_manager):
+            scope = '/subscriptions/{0}'\
+                .format(resource_manager.get_session().get_subscription_id())
+            return {'scope': scope}
+
+
+@CostManagementExport.filter_registry.register('last-execution')
+class CostManagementExportFilterLastExecution(Filter):
+    """ Find Cost Management Exports with last execution more than X days ago.
+
+    :example:
+
+    Returns all cost exports that didn't run in last 30 days.
+
+    .. code-block:: yaml
+
+        policies:
+          - name: find-stale-management-exports
+            resource: azure.cost-management-export
+            filters:
+              - type: last-execution
+                age: 30
+    """
+
+    schema = type_schema(
+        'last-execution',
+        required=['age'],
+        **{
+            'age': {'type': 'integer'}
+        }
+    )
+
+    def process(self, resources, event=None):
+        self.client = self.manager.get_client()
+        self.scope = 'subscriptions/{0}'.format(self.manager.get_session().get_subscription_id())
+        self.min_date = datetime.datetime.now() - datetime.timedelta(days=self.data['age'])
+
+        result, _ = ThreadHelper.execute_in_parallel(
+            resources=resources,
+            event=event,
+            execution_method=self._check_resources,
+            executor_factory=self.executor_factory,
+            log=log
+        )
+
+        return result
+
+    def _check_resources(self, resources, event):
+        result = []
+
+        for r in resources:
+            if gap('last-execution') in r:
+                continue
+            history = self.client.exports.get_execution_history(self.scope, r['name'])
+
+            # Include exports that has no execution history
+            if len(history.value) == 0:
+                r[gap('last-execution')] = 'None'
+                result.append(r)
+                continue
+
+            last_execution = max(history.value, key=lambda a: a.submitted_time)
+            if last_execution.submitted_time.date() <= self.min_date.date():
+                r[gap('last-execution')] = last_execution.serialize(True)
+                result.append(r)
+
+        return result
+
+
+@CostManagementExport.action_registry.register('execute')

```

Comment:
Should the verb here be `execute` or `export`?

I think `export` makes more sense to me, but I think we should just follow what the docs terminology. (If they exist).

@stefangordon  maybe you have an opinion here?

Reply:
I used API name.. afaik, this is not available in Az CLI now, but I'd expect them to follow API names as well

Reply:
Oh ok, yeah if that is the API name then it makes sense.  Probably because they use `Export` as the proper-noun name of an ARM resource so they have to have a different verb.